File: CS/PS/Site/leetcode/easy/1480.py
from typing import List
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
      for v in range(1, len(nums)):
        nums[v] = nums[v] + nums[v - 1]
      return nums
    runningSum(1,[0,1,2,3,4,5,6,7,8,9]);

File: CS/PS/Site/leetcode/easy/1480.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function (nums) {
  let answer = [];
  answer.push(nums[0]);
  nums.forEach((value, index) => { 
    
    if (index != 0)
      answer.push(value + answer[index - 1]);
  });
  return answer;
};



console.log(runningSum([1,2,3,4,5]));
